fxa/tests/test_requests_auth_plugin.py

```diff
@@ -81,6 +82,25 @@ def test_client_state_not_set_by_default(self, client_patch):
         r = auth(Request())
         self.assertNotIn('X-Client-State', r.headers)
 
+    @mock.patch('fxa.plugins.requests.core.Client',
+                return_value=mocked_core_client())
+    def test_memory_cache_is_set_by_default(self, client_patch):
+        auth = FxABrowserIDAuth(email="test@restmail.com",
+                                password="this is not a password",
+                                server_url="http://localhost:5000")
+        assert type(auth.cache) is MemoryCache
+
```

Comment:
Some missing assertions:

* cache is instantiated with default expiry
* cache is set with value on first call
* cache is used on second call